### PR TITLE
[Bug] fix reset offset value update issue on drawer in  consumer group table

### PR DESCRIPTION
**Pull request by ajaypratap003**
Partition Offset in UI is not updated after the offset has been reset
https://issues.redhat.com/browse/MGDSTRM-5206

image

### CODE REVIEW

**Comment by jgiardino** [comment]
Thanks for picking up this task, @ajaypratap003! I'm not able to read the code and fully understand what's happening. The question I have is whether you make a new api call to load fresh data, or if you're just passing state from the modal back to the page?

I assume we'd want to make a new api call, but just wanted to confirm that is the behavior you implemented.

I'd also like to add @wtrocki just to get another set of eyes on this one. 👀

File: src/modules/ConsumerGroups/ConsumerGroups.tsx [hunk 1]
```diff
@@ -79,6 +81,15 @@ const ConsumerGroups: React.FunctionComponent<ConsumerGroupsProps> = ({
     fetchConsumerGroups();
   }, [search, order]);
 
+  useEffect(() => {
+    //update setConsumerGroupDetail state after reset offset value
+    //so that values will update on page
+    const filteredGroup =
+      consumerGroups &&
+      consumerGroups.items?.filter((g) => g.groupId === groupId)[0];
```
Comment (wtrocki) [inline src/modules/ConsumerGroups/ConsumerGroups.tsx:89]: Adjusting array without checking size. 
Even if state would always have single item in array 
it is good to check size

**Comment by wtrocki** [comment]
Changes looks good. Have no means to verify it

**Comment by ajaypratap003** [comment]
> Thanks for picking up this task, @ajaypratap003! I'm not able to read the code and fully understand what's happening. The question I have is whether you make a new api call to load fresh data, or if you're just passing state from the modal back to the page?
> 
> I assume we'd want to make a new api call, but just wanted to confirm that is the behavior you implemented.
> 
> I'd also like to add @wtrocki just to get another set of eyes on this one. eyes

@jgiardino We had already api to refetch the data but we were missing to pass update data to  ResetOffset table.

**Comment by ajaypratap003** [comment]
@jgiardino @wtrocki Thanks for review PR and adding valuable feedback.

**Pull request merged**